operator/internal/manifests/internal/config/loki-runtime-config.yaml

```
@@ -58,8 +58,7 @@ overrides:
     blocked_queries:
     {{- range $blockedQuery := . }}
     - {{ with $blockedQuery.Pattern -}}
-      pattern: |
-        {{ . | yamlBlock "        "  }}
+      pattern: {{ . | yamlBlock "        "  }}
```

Thinking about the "let's just use a string instead of a YAML multi-line block" topic again, I think we can just replace this with the "quoted Go" `sprintf` output and remove the custom function:

```suggestion
      pattern: {{ . | printf "%q" }}
```